README.md

```diff
@@ -12,7 +12,7 @@
 
 ## Live Demo (if available)
```

Comment:
[OPTIONAL] Since it is available, i would suggest you remove the (if available).